### PR TITLE
added print schema command line option

### PR DESCRIPTION
**Pull request by joedevivo**
Added a commandline flag to reproduce this output: https://gist.github.com/joedevivo/7305410

### CODE REVIEW

File: src/cuttlefish_escript.erl [hunk 1]
```diff
@@ -226,6 +234,19 @@ zero_pad(Integer) ->
         _ -> [$0|S]
     end.
 
+print_schema(Schema) ->
+    lager:info("Outputing Schema Mappings"),
```
Comment (seancribbs) [inline src/cuttlefish_escript.erl]: "Printing"? :tongue:

**Comment by lenary** [comment]
Looks good to me, but I haven't run the code.

File: src/cuttlefish_escript.erl [hunk 2]
```diff
@@ -226,6 +234,19 @@ zero_pad(Integer) ->
...
+    {_, Mappings, _} = Schema,
+    X = [
+        {cuttlefish_mapping:mapping(M), string:join(cuttlefish_mapping:variable(M), ".")}
+    || M <- Mappings],
+
+    Max = lists:max([ length(Y) || {Y,_} <- X]),
```
Comment (seancribbs) [inline src/cuttlefish_escript.erl]: I know you know how much I love list-comprehensions, but you could reduce the number of times you iterate over this list by computing the max length as you extract the pairs by doing a `lists:foldr`.

**Comment by seancribbs** [comment]
Works for me when copied into a generated release and by tweaking the env.sh, but not standalone using all the appropriate flags (go figure?).  

Suggested improvements:
- Sort by the cuttlefish key or the translation key so that related settings are grouped together.
- Suppress lager output and don't generate output files when running with `-p`.

Otherwise, :+1:

File: src/cuttlefish_escript.erl (hunk 2 above)
Comment (joedevivo) [inline src/cuttlefish_escript.erl]: _sigh_ I wish I could only iterate once, but even with that case I have to iterate twice.

**Comment by joedevivo** [comment]
@seancribbs right now -p is designed to run during riak chkconfig which means adding a -p to env.sh would break riak start if we stopped generating app.configs.

Also, I wanted them to appear in the order they are defined, but maybe that's not the best. I'll sort it.

**Comment by joedevivo** [comment]
blah, that's another interator, I'm going back on my promise to sort it :P

**Pull request merged**